cmdx.py

```diff
@@ -1282,7 +1294,9 @@ def dagPath(self):
 
         """
 
-        return om.MDagPath.getAPathTo(self._mobject)
+        if self._mdagpath is None:
+            self._mdagpath = om.MDagPath.getAPathTo(self._mobject)
```

Comment:
Are we sure this won't break, if the node is later moved? The node reference to cmdx would be reused, but this path might break. We might need to compute this every time.

Reply:
I tested that behaviour it looks solid. I changed the hierarchy of these and everything still works fine (from str to matrices)

Caching MDagPath instances have huge benefits for castring components. I'll try some other tests just to be sure